README.rst

```diff
@@ -109,24 +109,24 @@ where type must be one of ``[build, ci, docs, feat, fix, perf, refactor, revert,
 Note: only `fix`, `feat`, and `perf` will trigger a new NPM release, as this is the default behavior for semantic-release.
 
 Versioning and releases
-*****
+***********************
 
 This library has its version automatically updated by Lerna (i.e., ``lerna version``) using semantic-versioning under-the-hood when the release is published to npm. Lerna is configured to use independent versioning with conventional commits, as opposed to keeping all package versions in sync.
 
-When a PR is merged, you must manually run Lerna to create a release commit (e.g., ``chore(release): publish``). In this commit, Lerna increments the versions in the appropriate package.json files for any changed packages, creates Git tags, and updates the CHANGELOG file.
-To create this commit:
-::
-
-    npm run lerna:version
-    git push --set-upstream origin automation/lerna/version
-    git push —-tags
+When a PR is merged, you must manually run ``lerna version`` to create a release commit (e.g., ``chore(release): publish new versions``). In this commit, Lerna increments the versions in the appropriate package.json files for any changed packages, creates Git tags, and updates the CHANGELOG file(s).
```

Comment:
minor suggestion: maybe don't include the ``lerna version`` thing here, and just say "you'll need to get lerna to create a release commit [...], steps are outlined below [...]. do the following after you merge your PR:"